src/taipy/core/_entity/_migrate.py

```diff
@@ -274,13 +275,21 @@ def _load_all_entities_from_fs(root: str = ".data") -> Dict:
     return entities
 
 
-def __write_entities_to_fs(_entities: Dict, root: str = ".data"):
+def __write_entities_to_fs(_entities: Dict, root: str):
```

Comment:
We should have as a default value the default value of the configuration.
What do you think?

Reply:
The default value should be processed at the CLI level.

I don't think we need/should have the default here.